client/src/pages/Greetings/Greetings.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { StyledDiv } from '../../assets/styles/styles';
+import Header from '../../components/Header/Header';
```

Comment:
I think you can change this import to just be `'../../components/Header'` if you add a `Header/index.ts` that has 
```typescript
export {default} from './Header'
```

It just removes the redundant component name in the import so it's a little nicer.

This goes for all other components, too